contrib/php-ssl.h

```diff
@@ -0,0 +1,33 @@
+/*
+  +----------------------------------------------------------------------+
+  | PHP Version 5                                                        |
+  +----------------------------------------------------------------------+
+  | Copyright (c) 1997-2015 The PHP Group                                |
+  +----------------------------------------------------------------------+
+  | This source file is subject to version 3.01 of the PHP license,      |
+  | that is bundled with this package in the file LICENSE, and is        |
+  | available through the world-wide-web at the following url:           |
+  | http://www.php.net/license/3_01.txt                                  |
+  | If you did not receive a copy of the PHP license and are unable to   |
+  | obtain it through the world-wide-web, please send a note to          |
+  | license@php.net so we can mail you a copy immediately.               |
+  +----------------------------------------------------------------------+
+  | Authors: Wez Furlong <wez@thebrainroom.com>                          |
+  |          Daniel Lowrey <rdlowrey@php.net>                            |
+  |          Chris Wright <daverandom@php.net>                           |
+  +----------------------------------------------------------------------+
+*/
+
+#ifndef MONGO_CONTRIB_SSL_H
+#define MONGO_CONTRIB_SSL_H
+
+#include "php.h"
+#include <openssl/x509.h>
+#include <openssl/x509v3.h>
```

Comment:
Doesn't this only work if the openssl headers are actually installed? Config.m4 only checks for SASL, and not openssl.